runtime/scripts.vim

```diff
@@ -306,7 +306,9 @@ else
     set ft=virata
 
     " Strace
-  elseif s:line1 =~# '^\(\[pid \d\+\] \)\=[0-9:.]* *execve(' || s:line1 =~# '^__libc_start_main'
+  elseif s:line1 =~# '^\(\[pid \d\+\] \)\=[0-9:.]* *execve('
+    \ || s:line1 =~# '^__libc_start_main'
+    \ || s:line1 =~# 'execve(.*)\s\+= \d\+'
```

Comment:
I might be missing something, but you only talk about 

> modern linux distributions

but don't give specific examples of what does not work. So please give a precise example (or show the strace generated logfile).
FYI, I just run `strace pstree 2>pstree.log && vim pstree.log` and it displays the logfile with strace syntax highlighting. (filetype being set to strace) For what is worth, the first line looks like this:
`execve("/usr/bin/pstree", ["pstree"], [/* 32 vars */]) = 0`

Also note, that `\=` basically means zero or one of the preceeding atom, so `^\(\[pid \d\+\] \)\=[0-9:.]* *execve(` basically means
`^` start of line followed by
`[pid <number>]` (but can be null) followed by
`[0-9:.]* *` (which also can be null) followed by
`execve(` which must appear.